codewit/client/index.html

```diff
@@ -2,15 +2,15 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <title>Client</title>
+    <title>Codewit.us</title>
```

Comment:
Let's stick with branding that's consistent with the (all lowercase) logo: `codewit.us`